components/service/glean/src/main/java/mozilla/components/service/glean/Glean.kt

```diff
@@ -88,6 +89,10 @@ open class GleanInternalAPI internal constructor () {
         applicationContext: Context,
         configuration: Configuration = Configuration()
     ) {
+        // In certain situations Glean.initialize may be called from a process other than the main
+        // process.  In this case we want initialize to be a no-op and just return.
+        if (!applicationContext.isMainProcess()) { return }
```

Comment:
Can we print a log line when this happens? (Is logging ok to call off the main process? I think so, but maybe @pocmo can confirm that)

Reply:
I think it's probably okay, I'll add it and test.